.env.example

```
@@ -1,136 +1,80 @@
+# Subscription payment links (from Stripe Dashboard → Payment Links)
+VITE_STRIPE_PAYMENT_LINK_YEARLY="https://buy.stripe.com/your-yearly-link-id"
+VITE_STRIPE_PAYMENT_LINK_MONTHLY="https://buy.stripe.com/your-monthly-link-id"
 
-# Subscription payment links
-VITE_STRIPE_PAYMENT_LINK_YEARLY="https://buy.stripe.com/your-yearly-link-id"  # 299€/year
-VITE_STRIPE_PAYMENT_LINK_MONTHLY="https://buy.stripe.com/your-monthly-link-id"  # 29€/month
-
-# Stripe webhook secret for payment verification (starts with whsec_)
-# Get this from Stripe Dashboard → Developers → Webhooks
+# Server-side secrets (from Stripe Dashboard → Developers)
+STRIPE_SECRET_KEY="sk_live_your-stripe-secret-key"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)